### PR TITLE
Band caterpillar: emphasize "show more projects"

### PR DESCRIPTION
**Pull request by modernserf**
Screen Shot 2019-03-15 at 1 45 33 PM

Add a fade-out gradient to the projects list. See https://glitch.manuscript.com/f/cases/3327693/

### CODE REVIEW

File: src/components/containers/expander.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (modernserf) [inline src/components/containers/expander.js]: - Is this component built correctly?
- Is this a reasonable category for this component?
- Should the Readme expander be refactored to use this component?

**Comment by keithk** [comment]
I like this a lot 👀

File: styles/profile.styl [hunk 2]
```diff
@@ -77,9 +77,9 @@ anonAvatar = "https://cdn.gomix.com/01a72eba-647e-4abf-abe6-89b02e388a7f%2Fanon-
     padding: 0
     margin-bottom: 1rem
     display: inline-block
-    overflow: visible
+    //overflow: visible
```
Comment (modernserf) [inline styles/profile.styl]: These were interfering with the container, and didnt appear to be doing anything useful. Am I missing something?

**Comment by modernserf** [comment]
Other design questions:
- should we add an additional row of visible projects?
- should we remove the count from the button?

File: src/components/containers/expander.js (hunk 1 above)
Comment (sheridanvk) [reply src/components/containers/expander.js]: Yes yes and yes! this all looks great :)

File: styles/profile.styl (hunk 2 above)
Comment (sheridanvk) [reply styles/profile.styl]: agree that this looks like it can be removed

**Pull request merged**